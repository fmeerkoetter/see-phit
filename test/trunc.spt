R"*(
<span>
  <p  color="re
)*"_html; 