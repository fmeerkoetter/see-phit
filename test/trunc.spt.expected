Error() [with int ROW = 4; int COL = 1; WHAT = spt::Unexpected_end_of_stream]':
